src/Vts.Test/MonteCarlo/Factories/DetectorFactoryTests.cs

```diff
@@ -139,13 +139,13 @@ public void Demonstrate_user_defined_detector_usage()
             };
             var simInput = new SimulationInput
                 {
-                    DetectorInputs = new[] { detectorInput },
+                    DetectorInputs = new IDetectorInput[] { detectorInput },

```

Comment:
I changed this here but it is suggesting this change elsewhere in this file, so I think I need to change it in a few more places. This is the reason behind the change:
https://www.jetbrains.com/help/resharper/2023.2/CoVariantArrayConversion.html